src/extras/index.js

```diff
@@ -8,5 +8,5 @@ export { default as TilingSpriteRenderer } from './webgl/TilingSpriteRenderer';
 export { default as BitmapText } from './BitmapText';
 
 export { default as cacheAsBitmap } from './cacheAsBitmap';
-export { default as getChildByName } from './getChildByName';
-export { default as getGlobalPosition } from './getGlobalPosition';
+import './getChildByName';
+import './getGlobalPosition';
```

Comment:
Can you add a comment that these just extend the prototype and don't have any exports?